engine/src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -54,7 +55,12 @@ object Backend {
 
   def callRootPathWithBaseRoot(jobDescriptor: BackendCallJobDescriptor, baseRoot: String): Path = {
     import io._
-    val call = s"$CallPrefix-${jobDescriptor.key.scope.unqualifiedName}"
+    import FinalCall._
```

Comment:
Maybe put the "enhancer" on the `descriptor` or `scope` rather than `descriptor.key.scope.name`?  It feels a bit strange to ask a name if it's a final call.